srcpkgs/ffmpeg/template

```diff
@@ -43,6 +43,7 @@ do_configure() {
 			aarch64*) _arch="aarch64";;
 			mips*) _arch="mips";;
 			ppc64*) _arch="ppc64";;
+			ppc*) _arch="ppc";;
```

Comment:
There is only 'ppc' and 'ppc-musl' this will be caught by the catch-all

Reply:
AAarch64 is also only two architectures, so why is it there?